setup.php

```diff
@@ -62,6 +62,14 @@ function plugin_init_escalade() {
                }
             }
 
+            // on ticket page (in edition)
+            if ((strpos($_SERVER['REQUEST_URI'], "ticket.form.php") !== false
+               || strpos($_SERVER['REQUEST_URI'], "problem.form.php") !== false
+               || strpos($_SERVER['REQUEST_URI'], "change.form.php") !== false) && isset($_GET['id'])) {
```

Comment:
globally ok with the pr.
But here, il woul be better to check if we really need to load the js file.
If none of hide buttons checkboxes enabled, don't load it